lib/index.js

```diff
@@ -322,17 +325,28 @@ api.update = brCallbackify(async ({actor, id, patch, sequence}) => {
   const record = await api.get({actor, id});
 
   if(record.meta.sequence !== sequence) {
-    return new BedrockError(
+    throw new BedrockError(
       'Could not update Account. Record sequence does not match.',
       'InvalidStateError', {
         httpStatusCode: 409,
         public: true,
         actual: sequence,
         expected: record.meta.sequence
-      });
+      }
+    );
   }
-
-  const errors = jsonpatch.validate(patch, record.account);
+  const customValidate = (operation, index, tree, existingPath) => {
+    jsonpatch.validator(operation, index, tree, existingPath);
+    const pathId = /\/id$/i.test(existingPath);
+    if(pathId) {
+      throw new jsonpatch.JsonPatchError(
+        `You can not change id ${existingPath}`,
```

Comment:
I think we typically don't say "You" in error messages, maybe something like this instead:

`"id" cannot be changed.`

Reply:
changing